static/css/student/profile.css

```diff
@@ -452,4 +452,88 @@ footer h6 {
 		width: 195px;
 		border-radius: 30px;
 	}
+}
+
+.course-card {
+	border-radius: 5px;
+	color: rgb(34,34,34);
+
+	margin: 10px;
+	height: 80px;
+}
+.course-card p{
+	font-size: 0.8em;
+	margin: 0;
+	display: inline-block;
+	width: 80%;
+}
+
+
+.red {
```

Comment:
are these semesterly colors? if so, we already have css declarations for them that we should reuse